runtime/doc/api.txt

```diff
@@ -3363,6 +3363,13 @@ nvim_create_autocmd({event}, {*opts})                  *nvim_create_autocmd()*
         })
 <
 
+    Example using a User command: >lua
```

Comment:
note: this file is autogenerated and this change has to be done to the docstring of `nvim_create_autocmd` in `src/nvim/api/autocmd.c`. Then use `make doc` to synchronize this file.

Reply:
Thanks for the heads up, fixed.